apps/projects/app/components/Content/Filters.js

```diff
@@ -79,42 +96,39 @@ export default class Filters extends Component {
     if (Object.keys(filterAliases).length === 0) return null
 
     return (
-      <div style={{
-        marginLeft: '8px',
-        marginTop: '10px',
-        flexDirection: 'row',
-        display: 'flex',
-        flex: '1',
-        flexWrap: 'wrap'
-      }}>
```

Comment:
The necessary change here was removing all of this styling expect for `display: flex`. This has been moved to the styled component called `Wrap`.

Reply:
All the other changes were either indentation-related or a result of running `prettier`.